server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -854,4 +858,48 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
 
     }
 
+    static class PeerRecoveryStats implements ToXContentFragment {
```

Comment:
A lot of content here is duplicated from `server/src/main/java/org/opensearch/indices/recovery/PeerRecoveryStats.java` - can we avoid this?